Projects/Playground/Playground.Forms.iOS/AppDelegate.cs

```diff
@@ -17,10 +17,5 @@ public override bool FinishedLaunching(UIApplication app, NSDictionary options)
 
             return base.FinishedLaunching(app, options);
         }
-
-        protected override MvxIosSetup CreateSetup(IMvxApplicationDelegate applicationDelegate, UIWindow window)
```

Comment:
I think this should remain, or find an alternative way for the application to define the setup class

Reply:
It has just gone from `abstract` to `virtual`. This makes it so much easier for beginners to start on Mvx.